src/shared/treePaths/useTreePaths.ts

```diff
@@ -1,42 +1,48 @@
 import dropRight from 'lodash/dropRight'
-import qs from 'qs'
+import qs, { ParsedQs } from 'qs'
 import { useMemo } from 'react'
 import { useLocation, useParams } from 'react-router-dom'
 
 import { useRepoOverview } from 'services/repo'
 import { getFilePathParts } from 'shared/utils/url'
 
-function getTreeLocation(paths, index) {
+function getTreeLocation(paths: string[], index: number) {
   return dropRight(paths, paths.length - index - 1).join('/')
 }
 
-export function useTreePaths(passedPath) {
+export function useTreePaths(passedPath?: string) {
   const location = useLocation()
 
-  const params = useMemo(() => {
+  const params: ParsedQs = useMemo(() => {
     return qs.parse(location.search, {
       ignoreQueryPrefix: true,
       depth: 1,
     })
   }, [location.search])
 
+  interface URLParams {
+    provider: string
+    owner: string
+    branch: string
+    path: string
+    repo: string
+    ref: string
+  }
+
   const {
     provider,
     owner,
     branch: urlBranch,
     path: urlPath,
     repo,
     ref: urlRef,
-  } = useParams()
+  } = useParams<URLParams>()
 
-  const { data: repoOverview } = useRepoOverview(
-    {
-      provider,
-      repo,
-      owner,
-    },
-    { suspense: false }
```

Comment:
was this removed on purpose?

Reply:
Yeah, `useRepoOverview` doesn't take that argument (and can't find where it ever did?) So removing this would be a no-op

Reply:
Got it! Nice fix :)